mrtt-ui/src/styles/theme.js

```diff
@@ -0,0 +1,31 @@
+import { createTheme } from '@material-ui/core/styles'
+// import { black } from '@mui/material/colors'
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#009B93'
```

Comment:
If we go with a css-in-js solution like emotion for this and want to use theming for any non-mui components, we might consider now or at some point having a more generic theme file that isnt MUI syntax that the MUI theme pulls stuff from. Im thinking it could take the name `theme.js`  and this one could be `muiTheme.js`.

For this stuff, it could just contain something like 

```
const colors = { weCanArgueAboutColourNamesLater: '#02B1A8', black: #000 }
export default {colors}

```